Increasing CRF range from 63 to 70
Changing CRF granularity from 1 to 0.25
diff --git a/libhb/common.c b/libhb/common.c
index dfd41edac..4cb7ef21e 100644
--- a/libhb/common.c
+++ b/libhb/common.c
@@ -1547,14 +1547,20 @@ void hb_video_quality_get_limits(uint32_t codec, float *low, float *high,
         case HB_VCODEC_FFMPEG_VP8:
         case HB_VCODEC_FFMPEG_VP9:
         case HB_VCODEC_FFMPEG_VP9_10BIT:
-        case HB_VCODEC_SVT_AV1:
-        case HB_VCODEC_SVT_AV1_10BIT:
             *direction   = 1;
             *granularity = 1.;
             *low         = 0.;
             *high        = 63.;
             break;
 
+        case HB_VCODEC_SVT_AV1:
+        case HB_VCODEC_SVT_AV1_10BIT:
+            *direction   = 1;
+            *granularity = 0.25;
+            *low         = 0.;
+            *high        = 70.;
+            break;
+
         case HB_VCODEC_VT_H264:
         case HB_VCODEC_VT_H265:
         case HB_VCODEC_VT_H265_10BIT:

The SVT-AV1-PSY fork store CRF/QP values in a particular way:
  - If we are using CRF values less than MAX_QP_VALUE (63) the integer
    part is stored in param->qp whereas the decimal part multiplied 
	by 4 is stored in param->extended_crf_quindex_offset
  - If we are using CRF values bigger than MAX_QP_VALUE we store 
    MAX_QP_VALUE inside param->qp and the remaining part inside
	param->extended_crf_qindex_offset (always multiplied by 4)
diff --git a/libhb/encsvtav1.c b/libhb/encsvtav1.c
index a2f18034f..195483b0c 100644
--- a/libhb/encsvtav1.c
+++ b/libhb/encsvtav1.c
@@ -25,6 +25,7 @@ int  encsvtWork(hb_work_object_t *, hb_buffer_t **, hb_buffer_t **);
 void encsvtClose(hb_work_object_t *);

 #define FRAME_INFO_SIZE 2048
+#define MAX_QP_VALUE 63
 #define FRAME_INFO_MASK (FRAME_INFO_SIZE - 1)

 hb_work_object_t hb_encsvtav1 =
@@ -124,7 +125,8 @@ int encsvtInit(hb_work_object_t *w, hb_job_t *job)
     }
     else
     {
-        param->qp                = job->vquality;
+        param->qp                         = fmin(job->vquality, MAX_QP_VALUE); // truncated 
+        param->extended_crf_qindex_offset = (job->vquality - param->qp) * 4;
         param->rate_control_mode = SVT_AV1_RC_MODE_CQP_OR_CRF;
         param->force_key_frames = 1;
     }
